### ISSUE TITLE
"Clean" dialogs shouldn't show an error

### ISSUE THREAD

**Issue by jeremydw** [blinkk/editor.dev-ui#131]
The red text makes it look like the user did something wrong, it shouldn't show an error until the user has tried to submit it and it is not valid.

![image](https://user-images.githubusercontent.com/646525/126734293-e663b800-8958-4f51-87c5-96134d483913.png)

**Comment by adelaidemt** [comment]
Wouldn't this only make sense if the "Add partial" was enabled whether or not the user selected an option? If it's disabled, there doesn't even need to be a red error message.

**Comment by jeremydw** [comment]
The issue is the user goes to add a partial, and immediately sees red. The error state should only be visible if the user tries to submit the form and the form is invalid (i.e. if it's missing a required field). The error states shouldn't appear by default.

**Comment by adelaidemt** [comment]
yes, i agree with that, but in the case that the "add partial" is
unclickable, then there wouldn't be an error message in the screenshot you
provided...only if the form required an input (not for radio fields).
Best,

Adelaide Taylor
Digital Producer | Atlanta, GA
***@***.***
